jinahub/crafters/TorchObjectDetectionSegmenter/README.md

```diff
@@ -0,0 +1,133 @@
+# ✨ TorchObjectDetectionSegmenter
+
+**TorchObjectDetectionSegmenter** is a class that supports object detection and bounding box extraction using PyTorch with Faster R-CNN and Mask R-CNN models.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [🌱 Prerequisites](#-prerequisites)
+- [🚀 Usages](#-usages)
+- [🎉️ Example](#%EF%B8%8F-example)
+- [🔍️ Reference](#%EF%B8%8F-reference)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## 🌱 Prerequisites
+
+To install the dependencies locally run 
+```
+pip install . 
+```
+
+To verify the installation works:
+```
+pip install -r tests/requirements.txt
+pytest -sv tests
+```
```

Comment:
We do not recommend user to use it in this way, right?

Reply:
yep, good catch. fixed